app/DynamicsAdapter/DynamicsAdapter.Web/PersonSearch/PersonSearchController.cs

```diff
@@ -166,5 +159,35 @@ public async Task<IActionResult> Rejected(Guid id, [FromBody]PersonSearchRejecte
 
             return Ok();
         }
+
+        private async Task<bool> UploadIdentifiers(Guid searchRequestId, PersonSearchCompleted personCompletedEvent, CancellationToken concellationToken)
+        {
+            if (personCompletedEvent.MatchedPerson.Identifiers == null) return true;
+            foreach (var matchFoundPersonId in personCompletedEvent.MatchedPerson.Identifiers)
+            {
+                SSG_Identifier identifier = _mapper.Map<SSG_Identifier>(matchFoundPersonId);
+                identifier.SSG_SearchRequest = new SSG_SearchRequest()
+                {
+                    SearchRequestId = searchRequestId
+                };
+                var identifer = await _searchRequestService.UploadIdentifier(identifier, concellationToken);
```

Comment:
I would change the signature of the methods to `AddIdentifier(Guid searchRequestId, SSG_SearchRequest searchRequest, CancelationToken toke`) technically it has more parameters which is not good, but it will put this in the method:
```
identifier.SSG_SearchRequest = new SSG_SearchRequest()
                {
                    SearchRequestId = searchRequestId
                };
```
so client of the methods don't have to set this before the call

Reply:
nice catch.